rules/klayout/drc/testing/run_switch_checking.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import os
+from subprocess import check_call
```

Comment:
do we actually want to exit on the first failure? or do we want to aggregate and exit with a compound status?